src/sass/extends/_heroes.scss

```diff
@@ -2,150 +2,289 @@
 # Heroes
 
 1. Mixins
-2. Hero, Desktop
-3. Hero, Mobile
-4. Story Hero, Mobile
-5. Story Hero, Desktop
+2. Hero, Mobile
+3. Hero, Tablet
+4. Hero, Desktop
+5. Story Hero, Mobile
+6. Story Hero, Desktop
```

Comment:
Yep, the hero markup and included CSS all changes with this addition. #armageddon :-)